docs/ABOUT.md

```diff
@@ -0,0 +1,14 @@
+Javascript is a scripting language used to provide dynamic and interactive content on webpages. It does not have class-oriented inheritance, but it does have prototype-oriented inheritance. Also, server side JS allows to use same language on server and client besides being fast. JavaScript provides benefits like :- 
+* Reducing server traffic by validating user input in the browser before it is sent to the server.
+* Providing immediate feedback to the site visitors so that they don't have to reload pages just to get error messages on form validations.
+* Allowing richer user interfaces with content changes on mouse hover, drag and drop gestures, and animations.
+
+Client-side JavaScript is interpreted in the browser without requiring compilation. This allows interactive content to be included in HTML pages which would otherwise be static.
+
+Server-Side JavaScript as run in NodeJS enables back-end access to databases, file systems, and servers. NodeJS is built on Google Chrome's JavaScript V8 Engine. NodeJS uses an event-driven, non-blocking I/O model that makes it lightweight and efficient. Node can be a great solution for applications requiring I/O bound operations, data streaming etc. More details can be found  [here](https://nodejs.org/en/about/)
+
+You should learn JavaScript because:- 
+* It's easy to learn.
+* It's versatile in the sense that its multi-paradigm supporting including procedural, event based, object oriented and functional programming.
```

Comment:
I may have made a mistake in my feedback here.  Since the diff is gone I can't tell, but this should probably read:

"sense that it's multi-paradigm..."  You want the apostrophe in that "it's".